pyproject.toml

```
@@ -10,8 +10,10 @@ readme = "README.md"
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = ">=3.11,<3.13"
+python = ">=3.10,<3.12"
```

If it's incompatible with 3.11, we need this to force 3.10.x, no?

```suggestion
python = ">=3.10,<3.11"
```

---

It is compatible with 3.11 if `pip install multiprocess=="0.70.15"` is run after poetry install.

---

Still relevant question?

---

Ah. So everything should be reverted to 3.11 then, no?